packages/admin/admin/src/common/buttons/clearinput/ClearInputButton.tsx

```diff
@@ -47,9 +56,8 @@ export function ClearInputButton(inProps: ClearInputButtonProps) {
 }
 
 /**
- * @deprecated Use `ClearInputAdornment` directly as the InputAdornment instead
+ * @deprecated Use `ClearInputAdornment` directly as the InputAdornment
  */
```

Comment:
I believe this was the original deprecation which was misplaced by accident. I'd suggest we remove this comment.

Reply:
Yes, but why remove the comment? 
It's still accurate, `ClearInputButton` has been replaced by `ClearInputAdornment`.

Reply:
Ah 😅